keps/sig-cli/3515-kubectl-explain-openapiv3/README.md

```diff
@@ -491,37 +456,30 @@ Defined using feature gate
 - Plaintext output roughly matches explain output
 - OpenAPIV3 (raw json) output implemented
 
-#### Alpha 2
-
-If we decide to move ahead with the `md` and `html` outputs, an Alpha 2 may
-be required.
-
-- `md` output implemented (or dropped from design due to continued debate)
-  - Table of contents all GVKs grouped by Group then Version.
-  - Section for each individual GVK
-  - All types hyperlink to specific section
--  basic `html` output  (or dropped from design due to continued debate)
-  - Table of contents all GVKs grouped by Group then Version.
-  - Page for each individual GVK.
-  - All types hyperlink to their specific page
-  - Searchable by name, description, field name.
-
 #### Beta
 
 - kube-openAPI v3 JSON deserialization is optimized to take less than 150ms on
   most machines
+    - No longer a goal for beta - we don't use kube-openapi types in this 
+      implementation. However a benchmark should still be written to measure
+      deserialization into the type used `map[string]any`
+
 - OpenAPI V3 is enabled by default on at least one version within kubectl's support window.
 As of Kubernetes 1.24 OpenAPIV3 entered beta and become enabled by default, therefore meeting this requirement.
+  - OpenAPI V3 has been Beta enabled by default since 1.24. Kubectl support window is +- 1 version. All clusters in kubectl's
+    support window have OpenAPI V3 enabled by default.
+
 - `--output plaintext` is on-by-default and environment variable is removed/on by default
 - `--output plaintext-openapiv2` added as a name for the old `explain` implementation, so the feature may be positively disabled.
+  - []TODO: Alias old implementation to plaintext-openapiv2
 
 #### GA
 
 - OpenAPIV3 is GA and has been since at least the minimum supported apiserver version
 by kubectl.
 - All kube-apiserver releases within version skew of kubectl should have OpenAPIV3 on by default. This is true as of kubectl for Kubernetes 1.25
```

Comment:
Yeah, not sure that's entirely convincing

Reply:
Yes this is a wrong statement since as you said elsewhere people can turn beta feature off. It is not safe to assume OPenAPIV3 is enabled until it is GA for all versions in skew

Reply:
changed requirement to require stable openapiv3